diff --git a/node_modules/react/cjs/react-jsx-dev-runtime.development.js b/node_modules/react/cjs/react-jsx-dev-runtime.development.js
index 585027f..aeca110 100644
--- a/node_modules/react/cjs/react-jsx-dev-runtime.development.js
+++ b/node_modules/react/cjs/react-jsx-dev-runtime.development.js
@@ -300,8 +300,9 @@
       ReactSharedInternals =
         React.__CLIENT_INTERNALS_DO_NOT_USE_OR_WARN_USERS_THEY_CANNOT_UPGRADE,
       hasOwnProperty = Object.prototype.hasOwnProperty,
-      isArrayImpl = Array.isArray,
-      createTask = console.createTask
+      isArrayImpl = Array.isArray;
+    new ("function" === typeof WeakMap ? WeakMap : Map)();
+    var createTask = console.createTask
         ? console.createTask
         : function () {
             return null;
diff --git a/node_modules/react/cjs/react-jsx-dev-runtime.react-server.development.js b/node_modules/react/cjs/react-jsx-dev-runtime.react-server.development.js
index 8d1141c..0d74219 100644
--- a/node_modules/react/cjs/react-jsx-dev-runtime.react-server.development.js
+++ b/node_modules/react/cjs/react-jsx-dev-runtime.react-server.development.js
@@ -304,8 +304,9 @@
         'The "react" package in this environment is not configured correctly. The "react-server" condition must be enabled in any environment that runs React Server Components.'
       );
     var hasOwnProperty = Object.prototype.hasOwnProperty,
-      isArrayImpl = Array.isArray,
-      createTask = console.createTask
+      isArrayImpl = Array.isArray;
+    new ("function" === typeof WeakMap ? WeakMap : Map)();
+    var createTask = console.createTask
         ? console.createTask
         : function () {
             return null;
diff --git a/node_modules/react/cjs/react-jsx-runtime.development.js b/node_modules/react/cjs/react-jsx-runtime.development.js
index 40bddcd..c18bba9 100644
--- a/node_modules/react/cjs/react-jsx-runtime.development.js
+++ b/node_modules/react/cjs/react-jsx-runtime.development.js
@@ -300,8 +300,9 @@
       ReactSharedInternals =
         React.__CLIENT_INTERNALS_DO_NOT_USE_OR_WARN_USERS_THEY_CANNOT_UPGRADE,
       hasOwnProperty = Object.prototype.hasOwnProperty,
-      isArrayImpl = Array.isArray,
-      createTask = console.createTask
+      isArrayImpl = Array.isArray;
+    new ("function" === typeof WeakMap ? WeakMap : Map)();
+    var createTask = console.createTask
         ? console.createTask
         : function () {
             return null;
diff --git a/node_modules/react/cjs/react-jsx-runtime.react-server.development.js b/node_modules/react/cjs/react-jsx-runtime.react-server.development.js
index dd07748..0b3aaf0 100644
--- a/node_modules/react/cjs/react-jsx-runtime.react-server.development.js
+++ b/node_modules/react/cjs/react-jsx-runtime.react-server.development.js
@@ -304,8 +304,9 @@
         'The "react" package in this environment is not configured correctly. The "react-server" condition must be enabled in any environment that runs React Server Components.'
       );
     var hasOwnProperty = Object.prototype.hasOwnProperty,
-      isArrayImpl = Array.isArray,
-      createTask = console.createTask
+      isArrayImpl = Array.isArray;
+    new ("function" === typeof WeakMap ? WeakMap : Map)();
+    var createTask = console.createTask
         ? console.createTask
         : function () {
             return null;
diff --git a/node_modules/react/cjs/react.development.js b/node_modules/react/cjs/react.development.js
index e3389a3..2bc8270 100644
--- a/node_modules/react/cjs/react.development.js
+++ b/node_modules/react/cjs/react.development.js
@@ -671,7 +671,9 @@
         getCurrentStack: null
       },
       hasOwnProperty = Object.prototype.hasOwnProperty,
-      createTask = console.createTask
+      REACT_CLIENT_REFERENCE = Symbol.for("react.client.reference");
+    new ("function" === typeof WeakMap ? WeakMap : Map)();
+    var createTask = console.createTask
         ? console.createTask
         : function () {
             return null;
@@ -681,7 +683,6 @@
     var didWarnAboutElementRef = {};
     var didWarnAboutMaps = !1,
       userProvidedKeyEscapeRegex = /\/+/g,
-      REACT_CLIENT_REFERENCE = Symbol.for("react.client.reference"),
       reportGlobalError =
         "function" === typeof reportError
           ? reportError
@@ -1276,7 +1277,7 @@
     exports.useTransition = function () {
       return resolveDispatcher().useTransition();
     };
-    exports.version = "19.1.0-experimental-f9d78089-20250306";
+    exports.version = "19.1.0-experimental-eda36a1c-20250228";
     "undefined" !== typeof __REACT_DEVTOOLS_GLOBAL_HOOK__ &&
       "function" ===
         typeof __REACT_DEVTOOLS_GLOBAL_HOOK__.registerInternalModuleStop &&
diff --git a/node_modules/react/cjs/react.production.js b/node_modules/react/cjs/react.production.js
index f357ca5..24d6e95 100644
--- a/node_modules/react/cjs/react.production.js
+++ b/node_modules/react/cjs/react.production.js
@@ -576,4 +576,4 @@ exports.useSyncExternalStore = function (
 exports.useTransition = function () {
   return ReactSharedInternals.H.useTransition();
 };
-exports.version = "19.1.0-experimental-f9d78089-20250306";
+exports.version = "19.1.0-experimental-eda36a1c-20250228";
diff --git a/node_modules/react/cjs/react.react-server.development.js b/node_modules/react/cjs/react.react-server.development.js
index 3cbd7eb..8497113 100644
--- a/node_modules/react/cjs/react.react-server.development.js
+++ b/node_modules/react/cjs/react.react-server.development.js
@@ -507,7 +507,9 @@
       REACT_CLIENT_REFERENCE$1 = Symbol.for("react.client.reference"),
       hasOwnProperty = Object.prototype.hasOwnProperty,
       assign = Object.assign,
-      createTask = console.createTask
+      REACT_CLIENT_REFERENCE = Symbol.for("react.client.reference");
+    new ("function" === typeof WeakMap ? WeakMap : Map)();
+    var createTask = console.createTask
         ? console.createTask
         : function () {
             return null;
@@ -517,7 +519,6 @@
     var didWarnAboutElementRef = {};
     var didWarnAboutMaps = !1,
       userProvidedKeyEscapeRegex = /\/+/g,
-      REACT_CLIENT_REFERENCE = Symbol.for("react.client.reference"),
       reportGlobalError =
         "function" === typeof reportError
           ? reportError
@@ -980,5 +981,5 @@
     exports.useMemo = function (create, deps) {
       return resolveDispatcher().useMemo(create, deps);
     };
-    exports.version = "19.1.0-experimental-f9d78089-20250306";
+    exports.version = "19.1.0-experimental-eda36a1c-20250228";
   })();
diff --git a/node_modules/react/cjs/react.react-server.production.js b/node_modules/react/cjs/react.react-server.production.js
index 53a5ddf..694169d 100644
--- a/node_modules/react/cjs/react.react-server.production.js
+++ b/node_modules/react/cjs/react.react-server.production.js
@@ -561,4 +561,4 @@ exports.useId = function () {
 exports.useMemo = function (create, deps) {
   return ReactSharedInternals.H.useMemo(create, deps);
 };
-exports.version = "19.1.0-experimental-f9d78089-20250306";
+exports.version = "19.1.0-experimental-eda36a1c-20250228";
diff --git a/node_modules/react/package.json b/node_modules/react/package.json
index 965519d..b6e85b0 100644
--- a/node_modules/react/package.json
+++ b/node_modules/react/package.json
@@ -4,7 +4,7 @@
   "keywords": [
     "react"
   ],
-  "version": "0.0.0-experimental-f9d78089-20250306",
+  "version": "0.0.0-experimental-eda36a1c-20250228",
   "homepage": "https://react.dev/",
   "bugs": "https://github.com/facebook/react/issues",
   "license": "MIT",
